.gitignore

```diff
@@ -27,4 +27,7 @@ client_ios/demo/demo.xcodeproj/project.xcworkspace/xcuserdata/*
 client_ios/demo/demo.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
 
 # artifacts zipfile
-*.zip
\ No newline at end of file
+*.zip
+
+# rust tools
+bacon.toml
```

Comment:
Looks cool: https://github.com/Canop/bacon

Reply:
yeah, was testing mostly for wasm workflow, but liked it overall. can just run `bacon check -- client --target wasm32-unknown-unknown`